modules/ROOT/pages/guided-setup.adoc

```diff
@@ -25,12 +25,54 @@ This may be the US control plane, the EU control plane, the MuleSoft Government
 +
 If your Anypoint account has the organization administrator role, the guided setup tool can create the connected app for you.
 +
-Otherwise, create the connected app or ask an organization administrator to create it, then provide the app details so the guided setup can perform the authentication and authorization.
+Otherwise, create the connected app or ask an organization administrator to create it with these steps:
++
+.. Create a key pair in Salesforce.
++
+This key pair is created in Salesforce and used for communication between Salesforce and Anypoint Platform.
```

Comment:
Consider combining so that we don't repeat "create in Salesforce."
Also, then we can end with a colon that points directly to the substeps (if you haven't found a way to promote them).

Reply:
Combined and added colon.